lib/compiler_rt/fixdfti.zig

```diff
@@ -1,12 +1,26 @@
+const builtin = @import("builtin");
+const arch = builtin.cpu.arch;
 const common = @import("./common.zig");
 const floatToInt = @import("./float_to_int.zig").floatToInt;
 
 pub const panic = common.panic;
 
 comptime {
-    @export(__fixdfti, .{ .name = "__fixdfti", .linkage = common.linkage });
+    const fixdfti_fn = if (builtin.os.tag == .windows and arch == .x86_64) b: {
+        // The "ti" functions must use Vector(2, u64) return types to adhere to the ABI
+        // that LLVM expects compiler-rt to have.
+        break :b __fixdfti_windows_x86_64;
+    } else __fixdfti;
+
+    @export(fixdfti_fn, .{ .name = "__fixdfti", .linkage = common.linkage });
```

Comment:
Suggest to follow the pattern of `want_aeabi `, `want_ppc_abi `, `gnu_f16_abi ` and make it look like this:

```suggestion
    if (common.windows_i128_abi) {
        @export(__fixdfti_windows_x86_64, .{ .name = "__fixdfti", .linkage = common.linkage });
    } else {
        @export(__fixdfti, .{ .name = "__fixdfti", .linkage = common.linkage });
    }
```

The comment body can go into the doc comments for `windows_i128_abi` in common.zig.